java/arcs/sdk/android/storage/service/StorageService.kt

```diff
@@ -228,6 +234,7 @@ class StorageService : ResurrectorService() {
 
     companion object {
         private const val EXTRA_OPTIONS = "storeOptions"
+        private const val MANAGER_ACTION = "MANAGER"
```

Comment:
It's typically a best-practice/pattern for most actions to include a package namespace, e.g. `arcs.sdk.android.storage.service.MANAGER`